docs/custom_quantum_functions.md

```diff
@@ -98,18 +98,25 @@ QMK provides methods to read 5 of the LEDs defined in the HID spec:
 * Compose
 * Kana
 
-There are two ways to get the host LED state:
+There are two ways to get the lock LED state:
 
 * by implementing `led_update_kb()` or `_user()`; or
 * by calling `host_keyboard_led_state()`
 
+!> `host_keyboard_led_state()` may already reflect a new value before `led_update_user()` is called.
+
+Two more deprecated functions exist that provide the LED state as a `uint8_t`:
+
+* `led_set_kb()` and `_user()`
```

Comment:
let's put the correct c signature in here. I would also like to mark the code. quantum.c (L1069) and quantum.h. I would add a TODO tag with a date to remove these calls or indicate they are now deprecated. People rarely read docs but they look at code.

Reply:
Done. I already added `\deprecated` doc comments.